electron/constants/index.js

```diff
@@ -17,7 +17,7 @@ const isProd = !isDev;
 // Paths
 const dotOperateDirectory = isProd
   ? path.join(os.homedir(), '.operate')
-  : '.operate';
+  : path.join(process.cwd(), '.operate');
```

Comment:
This change is required to support download of Tendermint. Otherwise, it will fail attempting to write on `/.operate/temp/`